### PR TITLE
firebase url fix

### PR DESCRIPTION
**Pull request by craigmulligan**
It was referencing firebaseapp.com but the datastore is on firebaseio.com.

### CODE REVIEW

**Comment by jviotti** [comment]
Looks good. I'll merge once the Hound CI comments are addressed.

File: app/scripts/app.coffee [hunk 1]
```diff
@@ -14,12 +14,11 @@ angular
...
+  .constant('URL', "#{window.location.protocol}//#{window.location.host.replace('firebaseapp', 'firebaseio')}/playlist")
   .config ($routeProvider) ->
     $routeProvider
       .when '/',
         templateUrl: 'views/main.html'
         controller: 'MainCtrl'
       .otherwise
-        redirectTo: '/'
-
+        redirectTo: '/'
```
Comment (craigmulligan) [inline app/scripts/app.coffee]: This seems consistent to me, also ran it through a linter? @jviotti

**Comment by jviotti** [comment]
@craig-mulligan Fixed the spaces issue myself. Do you mind squashing the five commits into one commit describing better what was fixed? I could do it myself but the commit will be under my name instead.

Make sure you pulled my spaces fix commit and do:

``` sh
$ git rebase -i HEAD~5
```

You will see something like:

``` sh
pick f126151 url fix
pick e04d8ec url fix
pick b9d788b url fix
pick da55569 linting
pick 5dca370 Fix spaces issue

# Rebase b374a7b..5dca370 onto b374a7b (5 command(s))
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#
# Note that empty commits are commented out
```

Change the commit list from:

``` sh
pick f126151 url fix
pick e04d8ec url fix
pick b9d788b url fix
pick da55569 linting
pick 5dca370 Fix spaces issue
```

To this (notice I set "squash" in all but the first commit):

``` sh
pick f126151 url fix
squash e04d8ec url fix
squash b9d788b url fix
squash da55569 linting
squash 5dca370 Fix spaces issue
```

Once you save the file, git will ask you to name the new commit. You can choose a more descriptive name there.

**Comment by jviotti** [comment]
@craig-mulligan Notice you will have to `git push --force` afterwards.

**Comment by craigmulligan** [comment]
@jviotti thanks for explaining rebases, super useful! Also notice I did a grunt build because the dist/scripts and styles were deleted in my previous commits.

**Comment by jviotti** [comment]
Looks great, merging. I also noticed that Hound CI is attempting to review your compiled JavaScript files. I'll disable that in another PR. Thanks!

**Pull request merged**